packages/keycloak-user-management/src/components/forms/UserForm/index.tsx

```diff
@@ -184,6 +188,15 @@ const UserForm: React.FC<UserFormProps> = (props: UserFormProps) => {
           >
             <Input disabled={initialValues.id ? true : false} />
           </Form.Item>
+          <Form.Item id="enabled" name="enabled" label="Enable user">
+            <Radio.Group name="enabled">
+              {status.map((e) => (
+                <Radio name="enabled" key={e.label} value={e.value}>
+                  {e.label}
+                </Radio>
+              ))}
```

Comment:
nitpick: you can pass status as `options` to `Radio.Group`, and have it automagically generate the options for you